### PR TITLE
win_virtio_driver_update: fix issue of reboot twice after downgrade driver

### PR DESCRIPTION
**Pull request by xiagao**
1. after driver downgrade test, vm is rebooted twice.
2. add enable driver verifier step.
ID: 1842760
Signed-off-by: Xiaoling Gao <xiagao@redhat.com>

### CODE REVIEW

**Comment by xiagao** [comment]
@lijinlijin Could you help review this patch? Thanks.

File: qemu/tests/win_virtio_driver_update_test.py [hunk 1]
```diff
@@ -39,13 +38,21 @@ def change_virtio_media(cdrom_virtio):
         logging.info("Changing virtio iso image to '%s'" % virtio_iso)
         vm.change_media("drive_virtio", virtio_iso)
 
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
```
Comment (lijinlijin) [inline qemu/tests/win_virtio_driver_update_test.py:42]: Do we still need it since you call "session = vm.wait_for_login(timeout=timeout)"?
kind of dup.

Comment (xiagao) [reply qemu/tests/win_virtio_driver_update_test.py:42]: I think it's okay to leave it here. 
it's verify vm's status.

Comment (vivianQizhu) [reply qemu/tests/win_virtio_driver_update_test.py:42]: @xiagao I agree with @lijinlijin It only wastes time, not necessary to verify it twice.

Comment (xiagao) [reply qemu/tests/win_virtio_driver_update_test.py]: update.

**Comment by vivianQizhu** [comment]
@lijinlijin Any more comments?

**Comment by lijinlijin** [comment]
No more comments.
Acked-by: Li Jin lijin@redhat.com

**Pull request merged**